my_style.css

```diff
@@ -95,5 +97,27 @@ nav a.active {
   background-color: lightblue;  
 }
 
+form {
+  background-color: #f9f9f9;
+  padding: 20px;
+  border-radius: 12px;
+  max-width: 600px;
+  margin: 30px auto;
+  box-shadow: 0 0 10px rgba(0,0,0,0.1);
+}
+
+fieldset {
+  border: 2px solid #0077cc;
+  border-radius: 8px;
+  padding: 15px;
+}
+
+legend {
+  font-weight: bold;
+  font-size: 1.2em;
+  color: #0077cc;
+}
+
+
```

Comment:
I had not noticed this before. In external CSS, you don't need the `<style>` tag. You can remove the first and last lines here.